Detectors/ITSMFT/ITS/simulation/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ O2_SETUP(NAME ${MODULE_NAME})
 set(SRCS
     src/V11Geometry.cxx
     src/V1Layer.cxx
```

Comment:
@sawenzel  There was kind of an additional consideration (Ruben might know it better)...
At some point, we may be asked to do simulations for so called "super-upgrade".  Because they are much less detailed, the old geometry classes (like V1Layer) are more easy to adapt for the super-upgrade purposes.  So, I would keep the old stuff in the repository.

However, true, we my consider removing the references to these classes from the CMakeLists, LinkDefs, etc.  For the moment at least.